src/main/java/cz/inventi/kpj/spring/springintroduction/persistence/GreetingRepository.java

```diff
@@ -1,5 +1,8 @@
 package cz.inventi.kpj.spring.springintroduction.persistence;
 
+import org.springframework.stereotype.Repository;
+
+@Repository
```

Comment:
The name of the service is confusing... You will learn about the repository in the next lesson. Now you should use the service but it will work.